docs/developer-docs/latest/update-migration-guides/migration-guides/v4/code/backend/content-type-schema.md

```diff
@@ -43,6 +43,8 @@ To convert Strapi v3 models to v4 content-types:
     ```sh
     mv src/api/<apiName>/models/ src/api/<apiName>/content-types/
     ```
+    
+::: tip If you have a lot of content types, you should maybe consider using [Strapi codemods](https://github.com/strapi/codemods/). This might save you a lot of time and allow you to migrate the file structure with that command line. 
```

Comment:
```suggestion
::: tip 
If you have a lot of content types, you should maybe consider using [Strapi codemods](https://github.com/strapi/codemods/). 
```

Thank you again @tonoli ! I updated the suggested text for brevity and better compliance with our [styleguide](https://handbook.strapi.io/user-success-manual/12-rules-of-technical-writing).

This PR is now approved, merged and will be live with the next deployment batch :-)